app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -1,19 +1,22 @@
 import React from "react";
 
 import { Alert, Toastr } from "neetoui";
+import { useTranslation } from "react-i18next";
+
+const DeleteAlert = ({ handleSubmit, isOpen, onClose }) => {
+  const { t } = useTranslation();
 
-const DeleteAlert = ({ handleDelete, isOpen, onClose }) => {
   const onSubmit = () => {
```

Comment:
Please use the `handle` prefix here. Please ensure that you don't make such mistakes in the future. Refer: https://academy.bigbinary.com/learn-react/event-handling/event-handlers

Reply:
Oh, my bad.
I misunderstood that.
Will make modification accordingly.